### DISCUSSION TITLE
<title>miniforge3

### DISCUSSION THREAD

**Discussion by Bassam-b** [community/community#61840] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

hello , i was try many time to setup the environment , in this point i get confused ( conda env create -f tensorflow-apple-metal.yml -n tensorflow ) , it contains all the packages inside pip and chanels everythig but not working

**Comment by BaraaBasim** [comment]
If you're trying to activate and environment using the conda activate <env name> command

**Reply by Bassam-b** [reply]
It didn't work
